src/vsr/replica.zig

```diff
@@ -4799,14 +4810,22 @@ pub fn ReplicaType(
         /// Construct a SV message, including attached headers from the current log_view.
         /// The caller owns the returned message, if any, which has exactly 1 reference.
         fn create_start_view_message(self: *Self, nonce: u128) *Message.StartView {
-            assert(self.status == .normal);
+            assert(self.status == .normal or self.status == .view_change);
             assert(self.syncing != .updating_checkpoint);
             assert(self.replica == self.primary_index(self.view));
-            assert(self.commit_min == self.commit_max);
             assert(self.commit_min <= self.op);
             assert(self.view >= self.view_durable());
             assert(self.log_view >= self.log_view_durable());
             assert(self.log_view == self.view);
+            if (self.status == .normal) {
+                assert(self.commit_min == self.commit_max);
+            } else {
+                // Potential primaries may send a SV message before committing up to commit_max.
+                // (see `repair`).
+                assert(self.status == .view_change);
+                assert(self.do_view_change_quorum);
+                assert(self.commit_max >= self.commit_min);
```

Comment:
Nit, but I would swap this to 
```suggestion
                assert(self.commit_min <= self.commit_max);
```
so that its for mirrors the similar assert in the preceding `status == .normal` branch.